### PR TITLE
Search: add chart.js dependency

### PR DESCRIPTION
**Pull request by bluefuton**
#### Changes proposed in this Pull Request:
Add chart.js dependency to the Search package. This is needed to render the Record Meter.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
Do `jetpack install --all` and ensure chart.js is installed.

### CODE REVIEW

**Review by kangzj — DISMISSED** [review]
It works and looks reasonable 👍

File: projects/packages/search/changelog/add-search-chart-js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: added
+
+Search: add chart.js package to dependencies
```
Comment (jsnmoon) [inline projects/packages/search/changelog/add-search-chart-js:4]: Nit: We probably don't need to bump the minor version for adding an unused dependency; perhaps a patch version bump would suffice?

**Review by jsnmoon — COMMENTED** [review]
Needs a merge/rebase. Happy to give it a :shipit: once conflicts are resolved!

**Review by jsnmoon — APPROVED** [review]
:shipit:

**Pull request merged**